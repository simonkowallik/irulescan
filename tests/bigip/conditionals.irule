if {"a" eq "a"} {
    expr 1
}
if {"a" equals "a"} {
    expr 2
}
if {"a" ends_with "a"} {
    expr 3
}
if {"a" starts_with "a"} {
    expr 4
}
if {"a" matches_glob "a"} {
    expr 5
}
if {"a" matches_regex "a"} {
    expr 6
}
if {"a" contains "a"} {
    expr 7
}
if {"a" and "a"} {
    expr 8
}
if {"a" or "a"} {
    expr 9
}
if { not (0) } {
    expr 10
}