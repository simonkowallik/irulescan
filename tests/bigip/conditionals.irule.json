[{"filepath":"conditionals.irule.json","warning":[],"dangerous":[]}]
