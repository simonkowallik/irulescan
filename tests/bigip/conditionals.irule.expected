WARNING: Unquoted expr at `1` in `expr 1`
WARNING: Unquoted expr at `2` in `expr 2`
WARNING: Unquoted expr at `3` in `expr 3`
WARNING: Unquoted expr at `4` in `expr 4`
WARNING: Unquoted expr at `5` in `expr 5`
WARNING: Unquoted expr at `6` in `expr 6`
WARNING: Unquoted expr at `7` in `expr 7`
WARNING: Unquoted expr at `8` in `expr 8`
WARNING: Unquoted expr at `9` in `expr 9`
WARNING: Unquoted expr at `10` in `expr 10`

